mycore-base/src/main/java/org/mycore/common/xml/MCRLayoutTransformerFactory.java

```diff
@@ -54,15 +51,13 @@
 
     private static Logger LOGGER = LogManager.getLogger(MCRLayoutTransformerFactory.class);
 
-    private static MCRFopper fopper = new MCRFopper();
-
     private static final MCRIdentityTransformer NOOP_TRANSFORMER = new MCRIdentityTransformer("text/xml", "xml");
 
     /**
      * Returns the transformer with the given ID. If the transformer is not instantiated yet,
      * it is created and initialized.
      */
-    public static MCRContentTransformer getTransformer(String id) throws Exception {
+    public MCRContentTransformer getTransformer(String id) throws Exception {
         MCRContentTransformer transformer = transformers.get(id);
         if (transformer != null) {
```

Comment:
It would be better if we use a ConcurrentHashMap and the method computeIfAbsent. It would make code much easier to understand and also multi threading proof.

Reply:
Target of this request was to separate the FO part without change the code and functionality. Better code should be solved in an other ticket.